Userland/Libraries/LibWeb/Infra/Strings.cpp

```diff
@@ -5,6 +5,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <AK/CharacterTypes.h>
```

Comment:
what's this for?

Reply:
Just an artifact from moving the original function a couple of times, will remove :)